### PR TITLE
feat: Added minimum element from stack at constant time

### PR DESCRIPTION
**Pull request by S-M-J-I**
#### Checklist

- [X] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [X] Added tests and example, test must pass
- [X] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [X] Relevant documentation/comments is changed or added
- [X] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [X] Search previous suggestions before making a new one, as yours may be a duplicate.
- [X] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -86,6 +86,7 @@
     "xtr1common": "cpp",
     "xtree": "cpp",
```
Comment (realstealthninja) [inline .vscode/settings.json:87]: remove modifications from this file thanks.

File: data_structures/min_ele_stack_const_time.cpp [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+ * Implementation of stack that retrieves the minimum element in constant time.
+ * (https://www.baeldung.com/cs/stack-constant-time)
+ *
+ * @details
+ * The MinStack data structure keeps track of the minimum element present in a
+ * stack and retrieves them at O(1) time.
+ */
+#include <cassert>
+#include <iostream>
+#include <stack>
```
Comment (realstealthninja) [inline data_structures/min_ele_stack_const_time.cpp:14]: document headers please
 https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: data_structures/min_ele_stack_const_time.cpp (hunk 2 above)
Comment (S-M-J-I) [reply data_structures/min_ele_stack_const_time.cpp]: @realstealthninja documented the headers

File: .vscode/settings.json (hunk 1 above)
Comment (S-M-J-I) [reply .vscode/settings.json]: @realstealthninja removed the modifications from this file.

File: data_structures/min_ele_stack_const_time.cpp (hunk 2 above)
Comment (S-M-J-I) [reply data_structures/min_ele_stack_const_time.cpp]: @realstealthninja merged the branches

**Pull request closed without merging**